sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -377,41 +377,42 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
     }
   }
 
-  test("CTAS a managed table with the existing empty directory") {
-    val tableLoc = new File(spark.sessionState.catalog.defaultTablePath(TableIdentifier("tab1")))
-    try {
+  protected def withTableLocation(tableNames: String)(f: File => Unit): Unit = {
```

Comment:
How about `withEmptyDirInTablePath(dirName: String)`?

Reply:
Ok,thank you !